pkg/security/probe/perf_buffer_monitor.go

```diff
@@ -433,7 +428,7 @@ func (pbm *PerfBufferMonitor) collectAndSendKernelStats(client statsd.ClientInte
 
 			// retrieve event type from key
 			evtType := model.EventType(id % uint32(model.MaxEventType))
-			tags[2] = fmt.Sprintf("event_type:%s", evtType)
+			tags := pbm.buildTags(fmt.Sprintf("map:%s", perfMapName), fmt.Sprintf("event_type:%s", evtType))
```

Comment:
Same here for the per map tag

Reply:
same thing, the event type is parsed fro, the id which is updated during each iteration ...